### PR TITLE
[TAS-1545] ✨ Add likerland nft metadata

### PR DESCRIPTION
**Pull request by williamchong**
Moved from liker-land api folder

### CODE REVIEW

File: src/routes/likerland/nft/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        'bookstore',
+      ].some((s) => selectedSet.has(s))
+    ) {
+      selectedSet.add('all');
+    }
+
+    const promises: Promise<any>[] = [];
+
+    if (['all', 'class_api', 'iscn'].some((s) => selectedSet.has(s))) {
+      promises.push(getNFTClassAndISCNMetadata(classId));
```
Comment (nwingt) [inline src/routes/likerland/nft/metadata.ts:239]:
```suggestion
      promises.push(...getNFTClassAndISCNMetadata(classId));
```

File: src/routes/likerland/nft/metadata.ts [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    } else if (selectedSet.has('class_chain')) {
+      promises.push(Promise.all([getNFTClassChainMetadata(classId), null]));
+    } else {
+      promises.push(Promise.resolve([null, null]));
+    }
+
+    if (['all', 'listing'].some((s) => selectedSet.has(s))) {
+      promises.push(getNFTClassOwnerInfoAndListingInfo(classId));
```
Comment (nwingt) [inline src/routes/likerland/nft/metadata.ts:247]:
```suggestion
      promises.push(...getNFTClassOwnerInfoAndListingInfo(classId));
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/routes/likerland/nft/metadata.ts (hunk 1 above)
Comment (williamchong) [reply src/routes/likerland/nft/metadata.ts:239]: `getNFTClassAndISCNMetadata` returns a `Promise<any[]>` which is not spreadable

File: src/routes/likerland/nft/metadata.ts (hunk 2 above)
Comment (williamchong) [reply src/routes/likerland/nft/metadata.ts:247]: `getNFTClassOwnerInfoAndListingInfo ` returns a `Promise<any[]>` which is not spreadable

**Pull request merged**